driver-core/src/main/java/com/datastax/driver/core/JdkSSLOptions.java

```diff
@@ -52,7 +53,12 @@ protected JdkSSLOptions(SSLContext context, String[] cipherSuites) {
 
     @Override
     public SslHandler newSSLHandler(SocketChannel channel) {
-        SSLEngine engine = newSSLEngine(channel);
+        return newSSLHandler(channel, null);
```

Comment:
Now that the class implements `RemoteEndpointAwareSSLOptions`, this method should never get called by our code. We could throw an `UnsupportedOperationException` or `AssertionError` here to ensure that this is the case.